docs/notary-server-config.md

```diff
@@ -0,0 +1,355 @@
+<!--[metadata]>
++++
+title = "Notary Serer Configuration File"
+description = "Specifies the configuration file for Notary Server"
+keywords = ["docker, notary, notary-server, configuration"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Notary Server Configuration File
+
+An example (full) server configuration file.
+
+```json
+{
+	"server": {
+		"addr": ":4443",
+		"tls_key_file": "./fixtures/notary-server.key",
+		"tls_cert_file": "./fixtures/notary-server.crt",
+		"auth": {
+			"type": "token",
+			"options": {
+				"realm": "https://auth.docker.io/token",
+				"service": "notary-server",
+				"issuer": "auth.docker.io",
+				"rootcertbundle": "/path/to/auth.docker.io/cert"
+			}
+		}
+	},
+	"trust_service": {
+		"type": "remote",
+		"hostname": "notarysigner",
+		"port": "7899",
+		"key_algorithm": "ecdsa",
+		"tls_ca_file": "./fixtures/root-ca.crt",
+		"tls_client_cert": "./fixtures/notary-server.crt",
+		"tls_client_key": "./fixtures/notary-server.key"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
+	},
+	"reporting": {
+		"bugsnag": "yes",
+		"bugsnag_api_key": "c9d60ae4c7e70c4b6c4ebd3e8056d2b8",
+		"bugsnag_release_stage": "notary-server"
+	}
+}
+```
+
+## `server` section (required)
+
+Example:
+
+```json
+"server": {
+	"addr": ":4443",
+	"tls_key_file": "./fixtures/notary-server.key",
+	"tls_cert_file": "./fixtures/notary-server.crt"
+}
+```
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Required</th>
+		<th>Description</th>
+	</tr>
+	<tr>
+		<td valign="top"><code>addr</code></td>
+		<td valign="top">yes</td>
+		<td valign="top">The TCP address (IP and port) to listen on.  Examples:
+			<ul>
+			<li><code>":4443"</code> means listen on port 4443 on all IPs (and
+				hence all interfaces, such as those listed when you run
+				<code>ifconfig</code>)</li>
+			<li><code>"127.0.0.1:4443"</code> means listen on port 4443 on
+				localhost only.  That means that the server will not be
+				acessible except locally (via SSH tunnel, or just on a local
+				terminal)</li>
+			</ul>
+		</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>tls_key_file</code></td>
+		<td valign="top">no</td>
+		<td valign="top">The path to the private key to use for
+			HTTPS.  Must be provided together with <code>tls_cert_file</code>,
+			or not at all. If neither are provided, the server will use HTTP
+			instead of HTTPS. The path is relative to the directory where
```

Comment:
This changes in https://github.com/docker/notary/pull/304 to be relative to the configuration file path, here and in `tls_cert_file`, `trust_server.tls_client_cert`, `trust_server.tls_client_key`, and `trust_server.tls_ca_file`.